tools/generate_kopernicus.cpp

```diff
@@ -57,10 +68,10 @@ std::map<std::string, std::string> const body_description_map = {
      "Requires envelope of volatiles in the form of oceans or ice."}};
 std::map<std::string, std::string> const body_name_map = {
     {"Trappist-1", "Sun"},
-    {"Trappist-1b", "Beta"},
-    {"Trappist-1c", "Charlie"},
+    {"Trappist-1b", "Charlie"},  // Not a typo, swap b and c.
+    {"Trappist-1c", "Beta"},
```

Comment:
Having discussed with @GregroxMun, let's not do that; Gregrox will endow b with an atmosphere and horrifying temperatures in a future update.

Reply:
Done.